docs/_components/banner.md

```diff
@@ -0,0 +1,172 @@
+---
+
+{% include helpers/base-component.html component="banner" %}
+
+## Default
+
+{% capture example %}
+<section
+  class="usa-banner"
+  aria-label="Official website of the United States government"
+>
+  <div class="usa-accordion">
+    <header class="usa-banner__header">
+      <div class="usa-banner__inner">
+        <div class="grid-col-auto">
+          <img
+            aria-hidden="true"
+            class="usa-banner__header-flag"
+            width="16"
+            height="11"
+            src="{{ site.baseurl }}/assets/img/us_flag.svg"
+            alt=""
+          />
+        </div>
+        <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
+          <p class="usa-banner__header-text">
+            An official website of the United States government
+          </p>
+          <p class="usa-banner__header-action">Here’s how you know</p>
+        </div>
+        <button
+          type="button"
+          class="usa-accordion__button usa-banner__button"
+          aria-expanded="false"
+          aria-controls="gov-banner-default"
+        >
+          <span class="usa-banner__button-text">Here’s how you know</span>
+        </button>
+      </div>
+    </header>
+    <div
+      class="usa-banner__content usa-accordion__content"
+      id="gov-banner-default"
+    >
+      <div class="grid-row grid-gap-lg">
+        <div class="usa-banner__guidance tablet:grid-col-6">
+          <img
+            class="usa-banner__icon usa-media-block__img"
+            src="{{ site.baseurl }}/assets/img/icon-dot-gov.svg"
+            role="img"
+            alt=""
+            aria-hidden="true"
+          />
+          <div class="usa-media-block__body">
+            <p>
+              <strong>Official websites use .gov</strong><br />A
+              <strong>.gov</strong> website belongs to an official government
+              organization in the United States.
+            </p>
+          </div>
+        </div>
+        <div class="usa-banner__guidance tablet:grid-col-6">
+          <img
+            class="usa-banner__icon usa-media-block__img"
+            src="{{ site.baseurl }}/assets/img/icon-https.svg"
+            role="img"
+            alt=""
+            aria-hidden="true"
+          />
+          <div class="usa-media-block__body">
+            <p>
+              <strong>Secure .gov websites use HTTPS</strong><br />
+              A <strong>lock</strong> (
+              <img src="{{ site.baseurl }}/assets/img/lock.svg" alt="Locked padlock icon" height="12" width="9" class="usa-banner__lock-image">
+              ) or <strong>https://</strong> means you’ve safely connected to
+              the .gov website. Share sensitive information only on official,
+              secure websites.
+            </p>
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+</section>
+{% endcapture %}
+{% include helpers/code-example.html code=example %}
+
+## Centered
+
+{% capture example %}
+<section
```

Comment:
this is a lot of repeated stuff for just the `id` and `class` to be different, do we ever consider making these into partials w/ variables?

```diff
2c2
<   class="usa-banner"
---
>   class="usa-banner usa-banner--centered"
28c28
<           aria-controls="gov-banner-default"
---
>           aria-controls="gov-banner-centered"
36c36
<       id="gov-banner-default"
---
>       id="gov-banner-centered"
```

Reply:
Yes! It was made difficult to propose with how I split up the pull requests, but part of what I'm hoping with #433 is to just `{% include banner.html %}` for these component examples and have some variable to create unique IDs.

Reflected in that PR message:

>This also opens the possibility of internally reusing "includes" for header with documented usage for guaranteed one-to-one documentation and usage

Reply:
I'll try to arrange merge order of the pull requests so that I can rebase to include this improvement.

Reply:
Rebased and updated to reuse `banner.html` in 4b3fcca.